src/Renderer/LayeredMaterial.js

```diff
@@ -76,7 +90,7 @@ var moveElementsArraySafe = function moveElementsArraySafe(array,index, howMany,
 };
 /* eslint-enable */
 
-const LayeredMaterial = function LayeredMaterial(id) {
+const LayeredMaterial = function LayeredMaterial(material, coord) {
```

Comment:
I think to keep WMTS logic.
it's more simple to work in same system (specially for non-linear systems)
What do you think?